src/loaders/agent.js

```diff
@@ -21,6 +21,10 @@ import { globalScope } from '../common/constants/runtime'
  * sensitive to network load, this may result in smaller builds with slightly lower performance impact.
  */
 export class Agent extends AgentBase {
+  /**
+   * @param {any} options Options to initialize agent with
```

Comment:
options has to be an object, no? Should description refer to readme for options?
```suggestion
   * @param {Object} options Options to initialize agent with
```